src/components/Toolbar/GradientGenerator.svelte

```
@@ -1,187 +1,155 @@
+                    };
+                    const removeListeners = () => {
+                        window.removeEventListener("mousemove", updateStopPos);
+                        window.removeEventListener("mouseup", removeListeners);
+                        isDraggingGradientHandler = false;
+                    };
+
+                    isDraggingGradientHandler = true;
+                    window.addEventListener("mousemove", updateStopPos);
+                    window.addEventListener("mouseup", removeListeners);
+                }}>
+            </div>
```

ESLint

[error] `<div>` with a mousedown handler must have an ARIA role(a11y_no_static_element_interactions)

(svelte/valid-compile)